apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/QueryOptions.cs

```diff
@@ -100,6 +100,12 @@ public sealed class QueryOptions
         /// </summary>
         public bool? DryRun { get; set; }
 
+        /// <summary>
+        /// The ID of the project in which to create the job. If this is not set,
```

Comment:
Why not? All these operations create a job, and the job is owned by a project. (So I might create a job in my project, even though it's querying public data.) I suspect in theory it's possible for three projects to be involved: "Create a job in project A to copy a table from project B to project C." (So long as the credentials allow that, of course.)

Any suggested wording that would get that idea across more clearly?